### PR TITLE
tests: add e2e for duplicating custom storage volume

### PR DESCRIPTION
**Pull request by mas-who**
## Done

- Add e2e tests for duplicating custom storage volume

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, [build and run as described in the docs](../CONTRIBUTING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Ensure CI passes

### CODE REVIEW

File: tests/storage.spec.ts [hunk 1]
```diff
@@ -153,3 +159,39 @@ test("storage volume of type block", async ({ page }) => {
 
   await deleteVolume(page, volume);
 });
+
+test("duplicate custom storage volume", async ({ page }) => {
+  const newProjectName = randomProjectName();
```
Comment (edlerd) [inline tests/storage.spec.ts:164]:
```suggestion
  const project = randomProjectName();
```

File: tests/storage.spec.ts [hunk 2]
```diff
@@ -153,3 +159,39 @@ test("storage volume of type block", async ({ page }) => {
...
+  const newPoolName = randomPoolName();
```
Comment (edlerd) [inline tests/storage.spec.ts:165]:
```suggestion
  const pool = randomPoolName();
```

File: tests/storage.spec.ts [hunk 3]
```diff
@@ -153,3 +159,39 @@ test("storage volume of type block", async ({ page }) => {
...
+  await createPool(page, newPoolName);
+  await createProject(page, newProjectName);
+
+  // duplicate volume in the same project and same pool
+  let newVolumeName = await duplicateVolume(page, volume);
```
Comment (edlerd) [inline tests/storage.spec.ts:170]:
```suggestion
  let duplicateVolume = await duplicateVolume(page, volume);
```

**Review by edlerd — APPROVED** [review]
LGTM, just think the variable names can be more speaking, so the test is easier to read.

File: tests/storage.spec.ts (hunk 3 above)
Comment (mas-who) [reply tests/storage.spec.ts:170]: the variable name and function name will clash, so I renamed the function name to `duplicateStorageVolume`.

**Review by edlerd — APPROVED** [review]
LGTM

**Pull request merged**